plugin.video.catchuptvandmore/resources/lib/channels/fr/francetv.py

```diff
@@ -32,390 +32,373 @@
 from resources.lib.menu_utils import item_post_treatment
+from resources.lib.addon_utils import get_item_media_path
 
 from kodi_six import xbmcplugin
 
+import re
 import json
 import time
 import urlquick
 
+from six.moves.html_parser import HTMLParser
+HTML_PARSER = HTMLParser()
+TAG_RE = re.compile(r'<[^>]+>')
+
 try:
     from itertools import zip_longest
 except ImportError:
     from itertools import izip_longest as zip_longest
+
 """
 Channels:
-    * France 2
-    * France 3 Nationale
-    * France 4
-    * France Ô
-    * France 5
-"""
-
-URL_API = utils.urljoin_partial("http://api-front.yatta.francetv.fr")
-
-URL_LIVE_JSON = URL_API("standard/edito/directs")
-# Find broadcast_Id (some mistmatchs on the json France 4 replace by the broacast Id of France 5)
-
-HEADERS_YATTA = {
-    'X-Algolia-API-Key': "80d9c91958fc448dd20042d399ebdf16",
-    'X-Algolia-Application-Id': "VWDLASHUFE"
-}
+    * france.tv (https://www.france.tv/)
 """
-URL_SEARCH_VIDEOS = "https://vwdlashufe-dsn.algolia.net/1/indexes/" \
-                    "yatta_prod_contents/query"
-
-URL_YATTA_VIDEO = URL_API + "/standard/publish/contents/%s"
-# Param : id_yatta
 
-URL_VIDEOS = URL_API + "/standard/publish/taxonomies/%s/contents"
-# program
-"""
+URL_API_MOBILE = utils.urljoin_partial("https://api-mobile.yatta.francetv.fr/")
+URL_API_FRONT = utils.urljoin_partial("http://api-front.yatta.francetv.fr")
 
 
 def replay_entry(plugin, item_id, **kwargs):
     """
     First executed function after replay_bridge
     """
-    return list_categories(plugin, item_id)
+    return francetv_root(plugin)
 
 
 @Route.register
-def list_categories(plugin, item_id, **kwargs):
-    """
-    Build categories listing
-    - actualités & société
-    - vie quotidienne
-    - jeux & divertissement
-    - ...
-    """
-    # URL example: http://api-front.yatta.francetv.fr/standard/publish/channels/france-2/categories/
-    # category_part_url example: actualites-et-societe
+def francetv_root(plugin):
 
-    url_categories = "standard/publish/channels/%s/categories/" % item_id
-    resp = urlquick.get(URL_API(url_categories))
-    json_parser = json.loads(resp.text)
-
-    for category in json_parser['result']:
-        item = Listitem()
-
-        item.label = category['label']
-        category_part_url = category['url']
+    # Channels
+    item = Listitem()
+    item.label = 'Chaînes'
+    item.set_callback(channels)
+    item_post_treatment(item)
+    yield item
 
-        item.set_callback(list_programs,
-                          item_id=item_id,
-                          category_part_url=category_part_url)
-        item_post_treatment(item)
-        yield item
-    """
-    Seems to be broken in current CodeQuick version
-    yield Listitem.recent(
-        list_videos_last,
-        item_id = item_id)
-    """
+    # Categories
     item = Listitem()
-    item.label = "Recent"
-    item.set_callback(list_videos_last, item_id=item_id)
+    item.label = 'Catégories'
```

Comment:
is this shown in the interface? If so it needs localization

Reply:
Yes it's only shown in the French TV section.
Does it needs a localization even if it's only in the French "section" of the add-on?

Reply:
Should be ok now 😉